### PR TITLE
Update metrics

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: data/stats.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-  terms: 1300
+  services: 712
+  terms: 1429
   analysis: 250
-  contributors: 28
-  reuses: 5
-  collections: 6
+  contributors: 49
+  reuses: 6
+  collections: 8
```
Comment (MattiSG) [inline data/stats.yaml:7]: > If it's easy to just get the count from the data, we might as well do that here 😇

File: data/stats.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 - date: 2023-05-01
-  services: 650
-  terms: 1300
+  services: 712
+  terms: 1429
```
Comment (MattiSG) [inline data/stats.yaml:3]: These are currently hardcoded with a `+`, so either remove the `+` from the template or round it down.
```suggestion
  services: 700
  terms: 1400
```

File: data/contributors.json [hunk 3]
```diff
@@ -44,6 +44,15 @@
     "Giacomo Stazi": "https://www.linkedin.com/in/giacomo-stazi-a14a15136/",
     "Jan Klass": "https://github.com/Kissaki",
     "Thomas Reboul": "https://www.linkedin.com/in/thomas-reboul-29a46569/",
-    "Harshil Jani": "https://github.com/Harshil-Jani"
+    "Harshil Jani": "https://github.com/Harshil-Jani",
+    "Sergei Kolesnikov": "https://github.com/smmkolesnikov",
+    "Emmanuel Haguet": "https://github.com/Manu1400",
```
Comment (MattiSG) [inline data/contributors.json:49]: Which contribution are we mentioning?

File: data/contributors.json [hunk 4]
```diff
@@ -44,6 +44,15 @@
...
+    "Gaspard Defréville": "https://github.com/GDefrevillePEReN",
+    "David Bertho": "https://github.com/dbertho",
+    "Pierre Mesure": "https://github.com/PierreMesure",
+    "Brice BAI": "https://github.com/bricebai1",
```
Comment (MattiSG) [inline data/contributors.json:53]:
```suggestion
    "Brice Bai": "https://github.com/bricebai1",
```

File: data/contributors.json [hunk 5]
```diff
@@ -44,6 +44,15 @@
...
+    "Sarah Mitko": "https://github.com/sarahmitko",
+    "Bertille Gauducheau": "https://github.com/Bertillegdch",
+    "omjdi": "https://github.com/omjdi"
```
Comment (MattiSG) [inline data/contributors.json:56]: Which contribution are we mentioning?

**Review by MattiSG — CHANGES_REQUESTED** [review]
The addition of new contributors, along with images and website, have to be confirmed with said contributors.

**Comment by Ndpnt** [comment]
> The addition of new contributors, along with images and website, have to be confirmed with said contributors.

OK, that's what I was wondering.

File: data/stats.yaml (hunk 1 above)
Comment (Ndpnt) [reply data/stats.yaml:7]: I do not understand what you mean here

**Review by clementbiron — APPROVED** [review]

**Review by MattiSG — APPROVED** [review]
👍 🚀

**Pull request merged**